ddg_api.py

```diff
@@ -7,27 +7,48 @@
 cors = CORS(app, resources={r"/*":
                             {"origins": "http://localhost:3000"}})
 
+global saved_mols
+# Can't serialize sets using json
```

Comment:
I don't understand this comment (why is it here?)

Reply:
left over from when guy was working on it, he left it as a reminder